README.md

```diff
@@ -47,8 +47,19 @@ Q . . . . . . .
 ```
 
 Alternatively, you can dump internal abstract syntax tree:
+example.c:
+```c
+struct {
+    int x; 
+    char y; 
+    struct { 
+        int t; 
+    } 
+    z; 
+} a;
+```
 ```shell
-echo 'struct {int x; char y; struct { int t; } z; } a;' | ./mzcc --dump-ast
```

Comment:
In gcc, specifying that a filename is `-`, the standard convention for saying 'standard input'.
Thus, both of the following lines are legal to generate executables:
```bash
echo 'int main() { return 0; }' | gcc -xc -O -Wall -o app /dev/stdin
echo 'int main() { return 0; }' | gcc -xc -O -Wall -o app -
```

It would be great if you can support the feature.

Reply:
That sounds good. I'll try that. Thanks

Reply:
Accepting source listing from stdin is important to implement test suite for verifying compilers. That is, we don't have to prepare lots of source files in advance.